src/bootstrap/dist.rs

```diff
@@ -1044,12 +1044,6 @@ impl Step for RustAnalyzer {
     }
 
     fn run(self, builder: &Builder<'_>) -> Option<GeneratedTarball> {
-        // This prevents rust-analyzer from being built for "dist" or "install"
-        // on the stable/beta channels. It is a nightly-only tool and should
-        // not be included.
-        if !builder.build.unstable_features() {
-            return None;
-        }
         let compiler = self.compiler;
         let target = self.target;
 
```

Comment:
There's a tarball.is_preview below that needs updating I think.

Reply:
I don't think so, but cause the actual component remains "*-preview" for the purpose of rustup updates, while the new rename is basically just an alias. We have the same thing still set on all formerly-preview components, even rustfmt.

Reply:
Ah, interesting. We should consider if there's a way to migrate, feels off that we need to keep what seems like the canonical name the same. Maybe the renames need to be in the other direction or something too?

Reply:
Maybe so, but we'll need to work with rustup folks to be sure the upgrade path works.